packages/e2e-tests/next-app-dynamic-routes/pages/optional-catch-all-ssg-with-fallback/[[...catch]].tsx

```diff
@@ -22,9 +22,15 @@ export default function OptionalCatchAllPage(
 
 export async function getStaticProps(
   ctx: GetServerSidePropsContext
-): Promise<{ props: OptionalCatchAllPageProps }> {
+): Promise<GetStaticPropsResult<OptionalCatchAllPageProps>> {
   const catchAll = ((ctx.params?.catch as string[]) ?? []).join("/");
 
+  if (catchAll === "not-found") {
+    return {
+      notFound: true
+    }
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/serverless-nextjs/serverless-next.js/pullRequest?prid=7710201)
```suggestion
    };
```